src/assets/wise5/components/openResponse/open-response-student/open-response-student.component.html

```diff
@@ -85,10 +85,9 @@
   </span>
   <div *ngFor="let attachment of attachments"
        fxLayout="row" fxLayoutAlign="start center">
-    <div *ngIf="attachment.type === 'audio'"
-        fxLayout="row" fxLayoutAlign="start center">
+    <div *ngIf="attachment.type === 'audio'" fxLayout="row" fxLayoutAlign="start center">
       <audio controls>
-        <source [src]="attachment.url | safeUrl" type="audio/webm">
+        <source src="{{attachment.url}}" type="audio/webm">
```

Comment:
Is there a reason why we can't use [src] here? I tried it and it seemed to work.